### PR TITLE
test log improvement

### LINKED ISSUE #678: Report containing the number of tests which have passed/failed
When running the tests, a report containing the tests which have failed and tests which have passed would be very useful.
Currently, in case of failure the report contains only the number of tests which have failed and in case of success the report contains only the number of tests which have passed. (please see the screenshot)
twotestsfailed

### PR DESCRIPTION
**Pull request by gituser173**
Fixes #678

Signed-off-by: Claudiu Ban claudiu-laurian.ban@hpe.com

### CODE REVIEW

File: cloudslang-content-verifier/src/main/java/io/cloudslang/lang/tools/build/SlangBuildMain.java [hunk 1]
```diff
@@ -160,28 +161,46 @@ private static Boolean parseCoverageArg(ApplicationArgs appArgs){
...
         log.info("------------------------------------------------------------");
         log.info("BUILD SUCCESS");
         log.info("------------------------------------------------------------");
         log.info("Found " + buildResults.getNumberOfCompiledSources()
                 + " slang files under directory: \"" + contentPath + "\" and all are valid.");
+        printNumberOfPassedSkippedTests(runTestsResults);
+        log.info("");
+    }
+
+    private static void printNumberOfPassedSkippedTests(RunTestsResults runTestsResults) {
```
Comment (Bonczidai) [inline cloudslang-content-verifier/src/main/java/io/cloudslang/lang/tools/build/SlangBuildMain.java]: a skipped test cannot be passed, right? :)

**Comment by Bonczidai** [comment]
:+1: 

@gituser173 please post a screenshot with the new output and highlight the changes - thx

**Comment by Bonczidai** [comment]
please also merge from master (sensitive + versioning)

**Comment by gituser173** [comment]
capture

**Comment by gituser173** [comment]
capture

**Comment by Bonczidai** [comment]
@andreeapuscas WDYT?

**Comment by andreeapuscas** [comment]
The improvement was the one expected.

**Pull request merged**